### DISCUSSION TITLE
Add a configuration repository for inspiration?

### DISCUSSION THREAD

**Discussion by no-mood** [nix-community/nixvim#1295] [Q&A] [answered]
Hi guys, I was starting to use nixvim (and nvim in general) and I was taking a look at the following [configuration](https://github.com/GaetanLepage/nix-config/tree/master/home/modules/tui/neovim).
Do you think it's a good idea to add a configuration repository, in the form of nix flakes, so that they can be grouped together and be used as inspiration?
Thank you very much

**Comment by daniellaing** [comment, marked as answer]
@GaetanLepage started one a couple of days ago at https://github.com/GaetanLepage/kickstart.nixvim
